src/Nox.Reference.Common/Extensions/MapperExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Reflection;
+using AutoMapper;
+
+namespace Nox.Reference.Common;
+
+public static class MapperHolder
```

Comment:
Why is this needed? I think we already have a mapper in World.Mapper field

Reply:
MapperHolder serves to register all included mapping. 
Because we use static instance of mapper we should register its mapping.

Reply:
let's discuss it